src/NServiceBus.Core/Pipeline/Outgoing/SendComponent.cs

```diff
@@ -0,0 +1,110 @@
+﻿namespace NServiceBus.Pipeline.Outgoing
+{
+    using System;
+    using Transport;
+
+    class SendComponent
+    {
+        SendComponent()
+        {
+        }
+
+        public static SendComponent Initialize(Configuration configuration, PipelineSettings pipelineSettings, HostingComponent hostingComponent, RoutingComponent routingComponent)
+        {
+            var newIdGenerator = GetIdStrategy(configuration);
+            pipelineSettings.Register("AttachCausationHeaders", new AttachCausationHeadersBehavior(newIdGenerator), "Adds related to and conversation id headers to outgoing messages");
+
+            pipelineSettings.Register(new AttachSenderRelatedInfoOnMessageBehavior(), "Makes sure that outgoing messages contains relevant info on the sending endpoint.");
+            pipelineSettings.Register("AuditHostInformation", new AuditHostInformationBehavior(hostingComponent.HostInformation, hostingComponent.EndpointName), "Adds audit host information");
+            pipelineSettings.Register("AddHostInfoHeaders", new AddHostInfoHeadersBehavior(hostingComponent.HostInformation, hostingComponent.EndpointName), "Adds host info headers to outgoing headers");
```

Comment:
It's odd to see host information related behaviors moved from the `HostingComponent` here.
What am I missing?

Reply:
the host doesn't care where it's information are being used. Especially it doesn't have a requirement to add these information to outgoing messages. This is a decision made by the outgoing pipeline.